incubator/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/streams/kafka/session.will.message.close.deliver.will.message/server.rpt

```diff
@@ -0,0 +1,339 @@
+                           .merged()
+                               .deferred(0)
+                               .partition(-1, -1)
+                               .key("client-1#will-signal")
+                               .header("type", "will-signal")
+                               .build()
+                           .build()}
+write ${mqtt:willSignal()
+            .clientId("client-1")
+            .delay(1)
+            .deliverAt(-1)
+            .lifetimeId("1")
+            .willId("1")
+        .build()}
+write flush
+
+# will signal for client-1, deliver at (now + delay)
+write await RECEIVED_WILL_DELIVER_AT_SIGNAL
+write zilla:data.ext ${kafka:dataEx()
+                           .typeId(zilla:id("kafka"))
+                           .merged()
+                               .deferred(0)
+                               .partition(-1, -1)
+                               .key("client-1#will-signal")
+                               .header("type", "will-signal")
+                               .build()
+                           .build()}
+write ${mqtt:willSignal()
+            .clientId("client-1")
+            .delay(1)
+            .deliverAt(2)
+            .lifetimeId("1")
+            .willId("1")
+        .build()}
+write flush
+
+#Once the will is delivered -> remove will message + signal
+read zilla:data.ext ${kafka:matchDataEx()
+                           .typeId(zilla:id("kafka"))
+                           .merged()
+                               .deferred(0)
+                               .partition(-1, -1)
+                               .key("client-1#will-signal")
+                               .hashKey("client-1")
+                               .header("type", "will-signal")
+                               .build()
+                           .build()}
+read zilla:data.null
+
+read zilla:data.ext ${kafka:matchDataEx()
+                           .typeId(zilla:id("kafka"))
+                           .merged()
+                               .deferred(0)
+                               .partition(-1, -1)
+                               .key("client-1#will")
+                               .hashKey("client-1")
+                               .build()
+                           .build()}
+read zilla:data.null
+
+# non-clean start -> fetch lifetimeId
+accepted
+
+read zilla:begin.ext ${kafka:matchBeginEx()
+                            .typeId(zilla:id("kafka"))
+                            .merged()
+                                .capabilities("FETCH_ONLY")
+                                .topic("mqtt_sessions")
+                                .filter()
+                                    .key("client-1#will-signal")
+                                    .build()
+                                .build()
+                            .build()}
+
+connected
+
+# no will signals
+write advise zilla:flush
+
+read closed
+write close
+
+
+# send initial migrate signal
```

Comment:
Change comment to `receive migrate signal from sender-1` and move after `connected`.